src/go/web/handlers.go

```diff
@@ -3332,6 +3338,214 @@ func GetError(w http.ResponseWriter, r *http.Request) {
 	w.Write(body)
 }
 
+// POST /experiments/{exp}/vms/{name}/mount
+func MountVM(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+
+	role := r.Context().Value("role").(rbac.Role)
+	if !role.Allowed("vms/mount", "patch") {
```

Comment:
Should use `post` instead of `patch` here since we're creating a new mount, and should pass `{exp_name}/{vm_name}` to `role.Allowed` as well to support per-VM access control for mounting.